### DISCUSSION TITLE
Any functional differences between Trivy as a container vs package?

### DISCUSSION THREAD

**Discussion by pdbrowne0001** [aquasecurity/trivy#1730] [Q&A]
Not counting the need for Docker, are there any functionality differences or limitations when running Trivy as a container?

**Comment by afdesk** [comment]
@pdbrowne0001 thanks for your interest in `trivy`.
Would you describe in more detail what limitation you mean?
for building trivy's docker image uses the same source code as for `trivy` tool, so this one contains the same features.

**Reply by pdbrowne0001** [reply]
I guess what I am trying to ask is are there any disadvantages to running Trivy as a container but I think you may have already answered my question with your response.
